java/client/src/main/java/glide/api/commands/GenericBaseCommands.java

```diff
@@ -542,4 +543,137 @@ CompletableFuture<Boolean> pexpireAt(
+     * @example
+     *     <pre>{@code
+     * client.lpush("mylist", new String[] {"3", "1", "2", "a"}).get();
+     * String[] payload = client.sort(
+     *      "mylist",
+     *      SortOptions.builder()
+     *          .alpha(true)
+     *          .orderBy(DESC)
+     *          .limit(new SortOptions.Limit(0L, 3L))
+     *          .build())
+     *      .get();
+     * assertArrayEquals(new String[] {"a", "3", "2"}, payload); // List is sorted in descending order lexicographically starting
+     * }</pre>
+     */
+    CompletableFuture<String[]> sort(String key, SortOptions sortOptions);
+
+    /**
+     * Sorts the elements in the list, set, or sorted set at <code>key</code> and returns the result.
+     * This command is routed depending on the client's <code>ReadFrom</code> strategy. The <code>
+     * sortReadOnly</code> command can be used to sort elements based on different criteria and apply
+     * transformations on sorted elements.
+     *
+     * @param key The key of the list, set, or sorted set to be sorted.
+     * @return A list of sorted elements.
+     * @example
+     *     <pre>{@code
+     * client.lpush("mylist", new String[] {"3", "1", "2"}).get();
+     * assertArrayEquals(new String[] {"1", "2", "3"}, client.sortReadOnly("mylist").get()); // List is sorted in ascending order
+     * }</pre>
+     */
+    CompletableFuture<String[]> sortReadOnly(String key);
+
+    /**
+     * Sorts the elements in the list, set, or sorted set at <code>key</code> and returns the result.
+     * This command is routed depending on the client's <code>ReadFrom</code> strategy. The <code>
+     * sortReadOnly</code> command can be used to sort elements based on different criteria and apply
+     * transformations on sorted elements.
+     *
+     * @param key The key of the list, set, or sorted set to be sorted.
+     * @param sortOptions The {@link SortOptions}.
+     * @return A list of sorted elements.
+     * @example
+     *     <pre>{@code
+     * client.lpush("mylist", new String[] {"3", "1", "2", "a"}).get();
+     * String[] payload = client.sortReadOnly(
+     *      "mylist",
+     *      SortOptions.builder()
+     *          .alpha(true)
+     *          .orderBy(DESC)
+     *          .limit(new SortOptions.Limit(0L, 3L))
+     *          .build())
+     *      .get();
+     * assertArrayEquals(new String[] {"a", "3", "2"}, payload); // List is sorted in descending order lexicographically starting
+     * }</pre>
+     */
+    CompletableFuture<String[]> sortReadOnly(String key, SortOptions sortOptions);
+
+    /**
+     * Sorts the elements in the list, set, or sorted set at <code>key</code> and stores the result in
+     * <code>destination</code>. The <code>sort</code> command can be used to sort elements based on
+     * different criteria, apply transformations on sorted elements, and store the result in a new
+     * key. To get the sort result without storing it into a key, see <code>sort</code>.
```

Comment:
```suggestion
     * key.<br>
     * To get the sort result without storing it into a key, see <code>sort</code>.
```
And add `{@link ...}`

Reply:
Please add such links everywhere you're referencing to another class/method